packages/hawtio/src/plugins/auth/oidc/oidc-service.ts

```diff
@@ -425,26 +473,47 @@ class OidcService implements IOidcService {
     }
   }
 
+  private oidcLoginFunction = (oidcIndex: number): (() => Promise<AuthenticationResult>) => {
+    log.info('GRGR', `Creating login function ${oidcIndex}`)
+    return () => {
+      log.info('GRGR', `Invoking login function ${oidcIndex}`)
```

Comment:
GRGR?
How about making the label more generic and changing the level to debug so that it can be kept in the codebase?

Reply:
oh my - I thought I removed these!!!! Sorry and thanks for your sharp eye ;)

that's my way to later find given code in browser JS debugger ;)